libtenzir/include/tenzir/fwd.hpp

```diff
@@ -506,7 +512,10 @@ CAF_BEGIN_TYPE_ID_BLOCK(tenzir_types, first_tenzir_type_id)
   // TODO: Make list, record, and map concrete typs to we don't need to do
   // these kinda things. See tenzir/aliases.hpp for their definitions.
   TENZIR_ADD_TYPE_ID((std::vector<tenzir::data>))
+  TENZIR_ADD_TYPE_ID((std::vector<tenzir::package>))
   TENZIR_ADD_TYPE_ID((tenzir::detail::stable_map<std::string, tenzir::data>))
+  TENZIR_ADD_TYPE_ID(
+    (std::vector<tenzir::detail::stable_map<std::string, tenzir::data>>))
```

Comment:
Is this a `vector<record>`? Where do we send that?

Reply:
In the context manager and in the package manager.